### ISSUE TITLE
PDFs of digitised items don't have non-text alternative 

### ISSUE THREAD

**Issue by LaurenFBaily** [wellcomecollection/wellcomecollection.org#12641]
### Background
Issue flagged in the Hassell Inclusion WCAG accessibility audit Oct 2025: 

**Downloading PDF shows a non-tagged export with no suitable non-text alternative** ([Non-text content](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html))

This refers to the downloadable item we make available for digitised books. This is a pdf containing the digitised images only. In the viewer we also provide the OCR text which can be accessed via a screenreader by those with a print disability. See example: https://wellcomecollection.org/works/a2239muq

### To do

- [ ] Meet with Digital Production and Collections Info colleagues to discuss print disability access provision.

### Definition of done:

- [ ] Decision has been made about whether we should be offering non-text alternatives to our digitised collections, the scope (provision for wider access?) & known challenges documented.
- [ ] Added to Digital Experience roadmap (and Digital Production backlog) if decision is to make a change
